android/assets/jsons/Civ V - Vanilla/Quests.json

```diff
@@ -0,0 +1,111 @@
+[
+  {
+    "name": "Route",
+    "description": "They want you to build a road to connect your capital to their city.",
```

Comment:
Quests should be written from the point of view of the city state, e.g. "Build a road to connect your capital to our city!"